### PR TITLE
[15.0][MIG] account_tax_group_widget_base_amount: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 15.0

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT36436

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
Code review and runboat test

File: account_tax_group_widget_base_amount/readme/ROADMAP.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+* With the refactor of the tax group widget in v15, hopefully this module won't be
+  necessary. As for the moment that change is still in developement, check when the
+  moment to migrate to such version arrives.
+  (See https://github.com/odoo/odoo/pull/63338#issuecomment-821035897)
```
Comment (chienandalu) [inline account_tax_group_widget_base_amount/readme/ROADMAP.rst]: Shouldn't this be updated as that debate was settled down already? Should it be proposed on localizations as the product owner suggested?

Comment (pedrobaeza) [reply account_tax_group_widget_base_amount/readme/ROADMAP.rst]: This is a general one, but Odoo has ignored this finally.

Comment (chienandalu) [reply account_tax_group_widget_base_amount/readme/ROADMAP.rst]: Ok, then this section of the readme should be updated in any case.

Comment (pedrobaeza) [reply account_tax_group_widget_base_amount/readme/ROADMAP.rst]: Yes, good point

**Review by chienandalu — APPROVED** [review]

**Comment by victoralmau** [comment]
Changes done

**Comment by chienandalu** [comment]
Let's go :)

**Pull request merged**